### PR TITLE
NTrep & CC Uniform Re-Sprite

### PR DESCRIPTION
**Pull request by Agameofscones**
Banishes the legacy sprites of yesteryear to the trashcan on my desktop. I also went about and cleaned up some type paths for all the CC uniforms (There were two redundant ones), added a new uniform, pulled some ranks out of my ass, assigned a new palette. 

Fun stuff that regular players can't access because admin content could use some love to.
(Except the NT Rep Uniform, that got changed as well.)

image
image
image
image
image
image

:cl: Triiodine
imageadd: 3x CC Uniform reworks.
imageadd: NTrep uniform rework. It is divided into a dress shirt & suit coat now.
imagedel: Legacy sprites for all CC Uniforms. No more flat black legacy sprites.
fix: Typepaths for all CC Uniforms are standardized.
del: Old (Redundant) CC Uniform definitions
/:cl:

### CODE REVIEW

**Comment by Ty-Omaha** [comment]
I like the black ones we have now better, not a fan of the green

**Comment by variableundefined** [comment]
I personally prefer the old sprites for NT Rep

**Comment by Xantholne** [comment]
Looks kinda charcoal-y than the deeper, sleeker black we have currently.

**Comment by Agameofscones** [comment]
If need be I can swap the palette from a deep sage back to black, however it won't look that great since I'd have to dip into the greys

**Comment by KasparoVy** [comment]
Glorious

**Comment by FreeStylaLT** [comment]
I like these! Admin CC uniforms having distinct shoulder-fluff is a great addition IMO.

**Comment by Ty-Omaha** [comment]
@FreeStylaLT We already have distinct shoulder fluff 
image

File: code/modules/clothing/under/jobs/civilian.dm [hunk 1]
```diff
@@ -113,15 +113,15 @@
...
-	item_state = "internalaffairs"
-	item_color = "internalaffairs"
+	icon_state = "ntrep"
+	item_state = "ntrep"
+	item_color = "ntrep"
 	flags_size = ONESIZEFITSALL
 
 /obj/item/clothing/under/rank/ntrep/skirt
-	desc = "A silky smooth black and gold representative uniform with blue markings."
+	desc = "A dress shirt and stark blue tie paired with a well tailored skirt. Smells like toner and the dry cleaner's."
```
Comment (Kyep) [inline code/modules/clothing/under/jobs/civilian.dm]: "cleaners".

File: code/modules/clothing/suits/jobs.dm [hunk 2]
```diff
@@ -278,7 +278,7 @@
 
 /obj/item/clothing/suit/storage/ntrep
 	name = "\improper Nanotrasen Representative jacket"
-	desc = "A fancy black jacket; standard issue to Nanotrasen Representatives."
+	desc = "A distinctively official looking deep green suit coat; standard issue to Nanotrasen Representatives."
```
Comment (Kyep) [inline code/modules/clothing/suits/jobs.dm]: Green?

File: code/modules/clothing/under/miscellaneous.dm [hunk 3]
```diff
@@ -88,43 +88,39 @@
...
+	displays_id = 0
+	flags_size = ONESIZEFITSALL
 
 /obj/item/clothing/under/rank/centcom/officer
-	desc = "Gold trim on space-black cloth, this uniform displays the rank of \"Lieutenant-Commander\" and bears \"N.C.V. Fearless CV-286\" on the left shoulder."
-	name = "\improper Nanotrasen Officers Uniform"
-	icon_state = "officer"
-	item_state = "g_suit"
-	item_color = "officer"
+	name = "\improper Nanotrasen Commanders Uniform"
```
Comment (Kyep) [inline code/modules/clothing/under/miscellaneous.dm]: "Commander's". As in, it belongs to the commander. Or perhaps "Commanders'" as in it belongs to the commanders, plural. I think. Maybe I'm wrong. Also, more generally there is no such rank as "Nanotrasen Commander" to the best of my knowledge.

File: code/modules/clothing/under/miscellaneous.dm [hunk 4]
```diff
@@ -88,43 +88,39 @@
...
-	item_state = "g_suit"
-	item_color = "officer"
-
-/obj/item/clothing/under/rank/centcom_commander
-	desc = "It's a jumpsuit worn by CentComm's highest-tier Commanders."
-	name = "\improper CentComm officer's jumpsuit"
-	icon_state = "centcom"
-	item_state = "dg_suit"
-	item_color = "centcom"
+/obj/item/clothing/under/rank/centcom/ranker
```
Comment (Kyep) [inline code/modules/clothing/under/miscellaneous.dm]: /ranker? If you're going to change the path, maybe make it something clear, like "/officer" or "/navyofficer".

**Comment by Agameofscones** [comment]
I wasn't aware that the Blueshield was given a new suit and or coat, I haven't been on the server in... 3 months now? So my view of what the hot new crew fashion is, _is_ a bit dated.

**Comment by Quantum-M** [comment]
This PR: https://github.com/ParadiseSS13/Paradise/pull/9875 , @Agameofscones

**Comment by variableundefined** [comment]
We've decided to put this to a staff vote to resolve the PR as the heads and maints cannot reach a conclusion on whether this should be merged or not. The result were overwhelmingly against this PR being merged, with some staff stating that this will be ok without the baggy pants, while others simply prefer the current sprite.

As a result this PR will be closed.

**Pull request closed without merging**